deploy/alerts/alerts.yaml

```diff
@@ -0,0 +1,136 @@
+
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  creationTimestamp: null
+  labels:
+    prometheus: example
+    role: alert-rules
+  name: prometheus-example-rules
```

Comment:
`name: stf-default-prometheus-openstack-rules` would be my suggestion.

Reply:
It works this way as well. Not sure if that's a bug or a feature. I'm happy to change it to `foo-bar-doesn-t-matter`

Reply:
I'm just trying to keep naming consistent across a deployment. My suggestions are generally how they would look should they have been deployed by the Operator using a template. It doesn't matter from a "does it work" perspective, but it does matter from a consistency and user experience perspective.

Reply:
Okay, the prometheus operator docs says, it's "best practice" to name it after the used prometheus. That apparently means, it doesn't matter. I'll rename it to follow best practices.